language.json

```diff
@@ -45,11 +45,11 @@
         "scanner": {
           "title": "Search for node or object",
           "search": "Search",
-          "noMatchingNodeOrObject": "No matching node or object for %{barcode}",
-          "noMatchingNode": "No matching node for %{uuid}",
-          "cannotActOnObject": "Can only act on barcode scans for objects from magasin or object picklist",
-          "cannotActOnNode": "Can only act on UUID scans from magasin, node picklist or move dialog",
-          "noCurrentLocation": "Object has no location"
+          "noMatchingNodeOrObject": "No matching node or object with barcode %{barcode}.",
+          "noMatchingNode": "No matching node with UUID %{uuid}.",
```

Comment:
Do we want to expose the user for the type UUID? Will it be better to change it to identifier or something similar?